climada/hazard/base.py

```diff
@@ -840,7 +840,7 @@ def year_month_day_accessor(
                 return array.dt.strftime("%Y-%m-%d").values
 
             # Handle access errors
-            except (ValueError, TypeError) as err:
+            except (ValueError, TypeError, AttributeError) as err:
```

Comment:
This is correct and expected. From the v2024.2 changelog:
> DataArray.dt now raises an AttributeError rather than a TypeError when the data isn’t datetime-like.
Thanks for the fix!